src/AdvancedMH.jl

```diff
@@ -19,7 +21,9 @@ export
     SymmetricRandomWalkProposal,
     Ensemble,
     StretchProposal,
-    MALA
+    MALA,
+    AMProposal,
+    AMSampler
```

Comment:
```suggestion
    AMProposal
```

This can be removed since we're not using `AMSampler` anymore.